openedx/core/djangolib/model_mixins.py

```diff
@@ -0,0 +1,14 @@
+"""
+Custom Django Model mixins.
+"""
+
+
+class DeprecatedModelMixin(object):
+    """
+    Used to make a class unusable in practice, but leave database tables intact.
+    """
+    def __init__(self, *args, **kwargs):
+        """
+        Override to kill usage of this model.
+        """
+        raise NotImplementedError("This model has been deprecated and should not be used.")
```

Comment:
Use a different exception.  NotImplementedError is for methods that need to be implemented by subclasses.  That's why you have to put pylint disables on all the model.  Make this raise a TypeError or something, and add a `__unicode__` method to it.  Then you won't need any of your disables.